File: broadcastreceiverlibrary/src/main/java/com/osapps/broadcastreceiverlibrary/sms/exceptions/SmsNoBundleException.java
package com.osapps.broadcastreceiverlibrary.sms.exceptions;

/**
 * Created by osapps on 09/05/2018.
 */
public class SmsNoBundleException extends SmsException {

    @Override
    protected String getErrorMsg() {
        return "sms bundle is null";
    }
}

File: broadcastreceiverlibrary/src/main/java/com/osapps/broadcastreceiverlibrary/sms/exceptions/SmsNoBundleException.kt
package com.osapps.broadcastreceiverlibrary.sms.exceptions

/**
 * Created by osapps on 09/05/2018.
 */
class SmsNoBundleException : SmsException() {

    override val errorMsg: String
        get() = "sms bundle is null"
}
